dbt/models/reporting/schema.yml

```diff
@@ -7,31 +7,31 @@ sources:
         description: '{{ doc("table_ratio_stats") }}'
         tests:
           - expression_is_true:
-              name: ratio_stats_no_nulls
-              expression: >-
+              name: reporting_ratio_stats_no_nulls
+              expression: |
                 year IS NOT NULL
-                  AND triad IS NOT NULL
-                  AND geography_type IS NOT NULL
-                  AND property_group IS NOT NULL
-                  AND assessment_stage IS NOT NULL
-                  AND sale_year IS NOT NULL
+                AND triad IS NOT NULL
+                AND geography_type IS NOT NULL
+                AND property_group IS NOT NULL
+                AND assessment_stage IS NOT NULL
+                AND sale_year IS NOT NULL
```

Comment:
All the deindenting and changes to `|` are just to get a cleaner compiled output i.e. one with line breaks.